doc/sphinx/conf.py

```diff
@@ -140,7 +140,14 @@ def setup(app):
 
 # If true, SmartyPants will be used to convert quotes and dashes to
 # typographically correct entities.
-#html_use_smartypants = True
+# Better to disable this to prevent alteration of command-line arguments
+# Sphinx <1.6 use `html_use_smartypants` and >=1.6 use `smartquotes`
+from sphinx import __version__ as sphinx_version
```

Comment:
[nitpick] I would have put the import at the top of the file together with the others, but YMMV